linux_sql/scripts/psql_docker.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
+
+#define the variables for the CLI arguments
+command=$1
+db_username=$2
+db_password=$3
+
+#start docker if docker server is not running
+sudo systemctl status docker || systemctl start docker
```

Comment:
make sure you understand how `||` works

Reply:
It's your responsibility to execute and test your program as I only review the coding style and high-level logic.